src/extension.ts

```diff
@@ -246,7 +246,7 @@ export async function activate(context: vscode.ExtensionContext) {
         if (usePipx) {
             let pipxList;
             try {
-                pipxList = JSON.parse(execFileSync("pipx", ["list", "--json"]).toString("utf8").trim());
+                pipxList = JSON.parse(execaSync("pipx", ["list", "--json"]).stdout.trim());
```

Comment:
This was not working on Windows. The problem is that the `pipx` executable on Windows is not named `pipx` but `pipx.cmd` and `execFileSync` cannot handle it. Decided to use the `execa` package to solve this issue.